AStarAlgorithmsProject/Alg Scripts/Point.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AStarAlgorithmsProject
+{
+    /// <summary>
+    /// Simple point class to make all our coding lives a little less of a hassle
+    /// Supplies the rules for finding distance between two points.
+    /// </summary>
+    class Point
+    {
+        private int x;
+        private int y;
+
+        public Point(int _x = 0, int _y = 0)
+        {
+            x = _x;
+            y = _y;
+        }
+
+        // Accessor Methods
+        public int X
+        { get { return x; } set { x = value; } }
```

Comment:
Set methods should be unnecessary.

Reply:
They could be? but say we use a point instance to hold onto and quickly change what our start tile is, you'd want to be able to change it then.

Reply:
We should just be creating a new point object when the starting location is changed. Making objects mutable that don't need to be is unnecessary imo.